Resources/Data Sets/index_nl.json

```diff
@@ -234,9 +234,9 @@
           "associated_datafile": "Invisibility Cloak.csv"
         },
         {
-          "name": "Keukenpapierrollen",
+          "name": "Keukenrollen",
```

Comment:
Niet gewoon "Keukenpapier" dat zit altijd in rollen toch?

Reply:
Niet een velletje

Reply:
Keukenrollen 

works for me. Keukenpapier sounds weird.

Reply:
Ja, keukenrollen